about.md

```diff
@@ -7,39 +7,47 @@ title: About Natron
 
 ![An animated cartoon shot open in Natron](img/about-screenshot.png)
 
-Natron is a free and open-source compositing application geared towards VFX work. It is currently actively developed by a small team of developers as a community effort.
+### Cross-platform open-source compositor for the visual effects industry
 
-We accept [feature requests and bug reports on GitHub](https://github.com/natronGitHub/natron/issues).
+Natron is a powerful free and open-source VFX compositor geared towards digital post-production work, with industry-standard UI, 100+ nodes, and support for OpenFX and community-made plugins. With its robust OpenFX based engine and support for OpenImageIO and OpenColorIO, Natron can handle all of your VFX post-production needs.
 
-## Development
-
-We regularly accept pull requests and are always looking for new contributors. If you are experienced in C++, Qt, and graphics programming, and are interested in contributing to Natron, please reach out to us!
+Natron was a proud recipient of Inria funding from 2013 to 2018, and is currently being developed by a team of open-source contributors as a community effort.
 
-For instructions on how to build Natron from sources please see [this section of our README document.](https://github.com/NatronGitHub/Natron#building-and-installing-from-source)
+{% include logo.html
```

Comment:
IMO this logo doesn't need to be here.  If we are including it, it should be white and an SVG.